Kickstarter-iOS/Views/Controllers/SettingsAccountViewController.swift

```diff
@@ -53,18 +71,38 @@ final class SettingsAccountViewController: UIViewController {
     _ = tableView
       |> settingsTableViewStyle
   }
+
+  func showCurrencyPickerCell(_ show: Bool) {
+    if show {
+      self.tableView.beginUpdates()
+      self.tableView.insertRows(at: [self.dataSource.insertCurrencyPickerRow()], with: .top)
+      self.tableView.endUpdates()
+    }
+  }
+
+  func dismissCurrencyPickerCell() {
+    if self.tableView.numberOfRows(inSection: SettingsAccountSectionType.payment.rawValue) == 3 {
```

Comment:
checking if section, `SettingsAccountSectionType.payment` is included in the section to then remove from from the section. This should be improved.